Source/WebInspectorUI/UserInterface/Controllers/JavaScriptLogViewController.js

```diff
@@ -368,17 +368,15 @@ WI.JavaScriptLogViewController = class JavaScriptLogViewController extends WI.Ob
             if (parentGroup)
                 this._currentSessionOrGroup = parentGroup;
         } else {
-            if (type === WI.ConsoleMessage.MessageType.StartGroup || type === WI.ConsoleMessage.MessageType.StartGroupCollapsed) {
+            if (this._addConsoleMessageToConsoleSourceMapErrorGroup(messageView)) {
+                return;
```

Comment:
This early return prevents the delegate method call  `this.delegate.didAppendConsoleMessageView(messageView);` on line 388 from being reached.